app/components/stepper-signup.js

```diff
@@ -0,0 +1,22 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { action, set } from '@ember/object';
+
+export default class StepperSignupComponent extends Component {
+  constructor() {
+    super(...arguments);
+
+    this.currentStep = Number(
+      +new URLSearchParams(window.location.search).get('step')
```

Comment:
why `+` here?

Reply:
using `+` to coerce a value into a number is not a good coding practice, kindly avoid. Use explicit coercion like `Number` constructor or `parseInt`

Reply:
Done with changes...can you please check my latest commit